lua/telescope/actions/generate.lua

```diff
@@ -19,7 +19,7 @@
 ---       },
 ---     },
 ---   }
---- </pre>
+--- </code>
```

Comment:
That's cool I didn't know this was a thing

Reply:
its a thing since yesterday :rofl: